custom_components/openhasp/__init__.py

```diff
@@ -184,26 +184,19 @@ async def async_setup(hass, config):
 
     hass.data[DOMAIN] = {CONF_PLATE: {}}
 
-    component = hass.data[DOMAIN][CONF_COMPONENT] = EntityComponent(
-        _LOGGER, DOMAIN, hass
-    )
+    component = hass.data[DOMAIN][CONF_COMPONENT] = EntityComponent(_LOGGER, DOMAIN, hass)
+
+    WAKEUP_SERVICE_SCHEMA = cv.make_entity_service_schema({})
+    PAGE_NEXT_SERVICE_SCHEMA = cv.make_entity_service_schema({})
+    PAGE_PREV_SERVICE_SCHEMA = cv.make_entity_service_schema({})
+
```

Comment:
I think you should also include the other services , just to maintain everything coherent